.github/workflows/canary-integration-test.yml

```diff
@@ -86,7 +86,7 @@ jobs:
         sed -i 's/<OSD-IDs>/1/' deploy/examples/osd-purge.yaml
         # the CI must force the deletion since we use replica 1 on 2 OSDs
         sed -i 's/false/true/' deploy/examples/osd-purge.yaml
-        sed -i 's|rook/ceph:v1.8.1|rook/ceph:local-build|' deploy/examples/osd-purge.yaml
+        sed -i 's|rook/ceph:v1.8.2|rook/ceph:local-build|' deploy/examples/osd-purge.yaml
```

Comment:
At some point, we should remove this and use a regex so that we don't have to modify the line every time :)

Reply:
Agreed!